runtime/j9vm/jvm.c

```diff
@@ -1496,24 +1506,131 @@ static jint initializeReflectionGlobals(JNIEnv * env, BOOLEAN includeAccessors)
 	return JNI_OK;
 }
 
-
-
 /**
- *  jint JNICALL JNI_CreateJavaVM(JavaVM **pvm, void **penv, void *vm_args)
- *  Load and initialize a virtual machine instance.
- *	This provides an invocation API that runs the J9 VM in BFU/sidecar mode
+ * jint JNICALL JNI_CreateJavaVM(JavaVM **pvm, void **penv, void *vm_args)
+ * Load and initialize a virtual machine instance.
+ * This provides an invocation API that runs the J9 VM in BFU/sidecar mode.
  *
- *  @param pvm pointer to the location where the JavaVM interface
+ * @param pvm pointer to the location where the JavaVM interface
  *			pointer will be placed
- *  @param penv pointer to the location where the JNIEnv interface
+ * @param penv pointer to the location where the JNIEnv interface
  *			pointer for the main thread will be placed
- *	@param vm_args java virtual machine initialization arguments
+ * @param vm_args java virtual machine initialization arguments
  *
- *  @returns zero on success; otherwise, return a negative number
+ * @returns zero on success; otherwise, return a negative number
  *
- *	DLL: jvm
+ * DLL: jvm
  */
 jint JNICALL JNI_CreateJavaVM(JavaVM **pvm, void **penv, void *vm_args) {
+	return JNI_CreateJavaVM_impl(pvm, penv, vm_args, FALSE);
+}
+
+#if defined(JITSERVER_SUPPORT)
+int32_t JNICALL
+JITServer_CreateServer(JITServer **jitServer, void *serverArgs)
```

Comment:
This should have a Doxygen style comment as well

Reply:
The comment is present in `oti/jitserver_api.h`.